### PR TITLE
Adds deploy

### LINKED ISSUE #5: Setup deploy

### PR DESCRIPTION
**Pull request by justinlittman**
closes #5

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
```
Comment (jcoyne) [inline config/secrets.yml]: Yeah, don't check this in.

File: config/secrets.yml [hunk 2]
```diff
@@ -1,4 +1,22 @@
+# Be sure to restart your server when you modify this file.
```
Comment (jcoyne) [inline config/secrets.yml]: Please remove this from the PR.

File: config/secrets.yml [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
```
Comment (jcoyne) [inline config/secrets.yml]: Why do we need these?

Comment (justinlittman) [reply config/secrets.yml]: Removed.

File: config/secrets.yml (hunk 2 above)
Comment (justinlittman) [reply config/secrets.yml]: Removed.

File: config/application.rb [hunk 4]
```diff
@@ -44,8 +44,11 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    # accept_request_filter omits OKComputer and Sidekiq
+    accept_proc = proc { |request| request.path.start_with?('/v1') }
```
Comment (jcoyne) [inline config/application.rb]: Nice!

Comment (mjgiarlo) [reply config/application.rb]: Great find, @justinlittman!

**Pull request merged**